demo-simple/src/main/res/layout/activity_main.xml

```diff
@@ -49,12 +49,6 @@
       android:layout_height="wrap_content"
       android:text="Migrate" />
 
-    <Button
```

Comment:
Removing this from the UI we are implicitly unbinding now. If wanting to test this can be done via `adb` or through the device directly by terminating a service.